### ISSUE TITLE
[minor bug] amplify delete on uninitialized repo still hard crashing in v4.21.3

### ISSUE THREAD

**Issue by sw-yx** [aws-amplify/amplify-cli#4632]
FYI @attilah problem still there in v4.21.3 
![image](https://user-images.githubusercontent.com/6764957/85305027-0f8fa400-b4df-11ea-96b8-4f5ea47c37e2.png)

was the fix released properly?

_Originally posted by @sw-yx in https://github.com/aws-amplify/amplify-cli/issues/4549#issuecomment-647586308_

**Comment by sw-yx** [comment]
here's my tree @brene 

```
amplify/
└── backend
    ├── api
    │   └── reactdraw
    │       ├── parameters.json
    │       ├── schema.graphql
    │       └── stacks
    │           └── CustomResources.json
    └── backend-config.json

4 directories, 4 files

```

**Comment by attilah** [comment]
No those fixes were applied after 4.1.3 release will be in the next release.

**Comment by SwaySway** [comment]
This should be fixed in the latest release (4.22 as of now). Should you run into this issue again please let us know.
